File: exam/java classes/Airplane.java
package com.example.exam;

public class Airplane {
    private String name;
    private String model;

    public Airplane(String name, String model) {
        this.name = name;
        this.model = model;
    }

    public Airplane() {

    }

    public String getName() {
        return name;
    }

    public String getModel() {
        return model;
    }

    @Override
    public String toString() {
        return name + ", " + model;
    }

    public void setName(String newName) {
        name = newName;
    }
}

File: mega test/mega test/Airplane.cpp
#include "Airplane.h"
#include<string>

Airplane::Airplane() : model(""), planeNumber("") {}
Airplane::Airplane( std::string model,  std::string planeNumber) :
	model(model), planeNumber(planeNumber) {}

std::string Airplane::GetModel() {
	return model;
}
std::string Airplane::GetPlaneNumber()  {
	return planeNumber;
}

void Airplane::SetModel( std::string model) {
	this->model = model;
}

void Airplane::SetPlaneNumber( std::string planeNumber) {
	this->planeNumber = planeNumber;
}


